test/functional/page_objects/settings_page.js

```diff
@@ -564,12 +564,8 @@ export function SettingsPageProvider({ getService, getPageObjects }) {
       log.debug(`Clicking importObjects`);
       await testSubjects.click('importObjects');
       log.debug(`Setting the path on the file input`);
-      if (browser.isW3CEnabled) {
-        const input = await find.byCssSelector('.euiFilePicker__input');
-        await input.type(path);
-      } else {
-        await find.setValue('.euiFilePicker__input', path);
-      }
+      const input = await find.byCssSelector('.euiFilePicker__input');
+      await input.type(path);
```

Comment:
Chromedriver release notes has an item: `Resolved issue 2994: Clicking on file upload does not error [Pri-2]`

`find.setValue` is doing click on input and it was causing a failure; Now the same code works everywhere.